src/js/node/http.ts

```diff
@@ -637,6 +647,7 @@ Server.prototype = {
           const prevIsNextIncomingMessageHTTPS = isNextIncomingMessageHTTPS;
           isNextIncomingMessageHTTPS = isHTTPS;
           const http_req = new RequestClass(req);
+          req.signal.addEventListener("abort", onRequestAborted.bind(http_req));
```

Comment:
I think we can be a little smarter about this

Reply:
Should I add a internal method that we just add the JSValue of the onRequestedAborted function on the request? we already have the request weak ptr to trigger this function with should be enough because RequestClass holds a strong ref to the request. We would increase the size of all `Request` objects by a little bit, but would avoid creating the abort signal it self.

In the perfect scenario we should just rewrote http module to use directly the C api and avoid all this wrapping.

Reply:
https://github.com/oven-sh/bun/pull/13738/commits/89aaed958953bbe688803fefa374f9c3807a9706